components/prism-nevod.js

```diff
@@ -0,0 +1,112 @@
+Prism.languages.nevod = {
+	'comment': /\/\/.*|(?:\/\*[\s\S]*?(?:\*\/|$))/,
+	'string': {
+		pattern: /(?:"(?:""|[^"])*"(?!")|'(?:''|[^'])*'(?!'))!?\*?/,
+		greedy: true,
+		inside: {
+			'string-attrs': /!$|!\*$|\*$/,
+		},
+	},
+	'namespace': {
+		pattern: /(@namespace\s+)[a-zA-Z0-9\-.]+(?=\s*{)/,
+		lookbehind: true,
+	},
+	'pattern': {
+		pattern: /(@pattern\s+)?#?[a-zA-Z0-9\-.]+(?:\s*(?:[(]\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]+\s*(?:,\s*(?:(?:~\s*)?[a-zA-Z0-9\-.]*))*)[)]))?(?=\s*=)/,
+		lookbehind: true,
+		inside: {
+			'pattern-name': {
+				pattern: /^#?[a-zA-Z0-9\-.]+/,
+				alias: 'class-name',
+			},
+			'attributes': {
+				pattern: /\(.*\)/,
+				inside: {
+					'attr-name': {
+						pattern: /[a-zA-Z0-9\-.]+/,
+						alias: 'variable',
+					},
+					'punctuation': /[,()]/,
+					'operator': {
+						pattern: /~/,
+						alias: 'attr-hidden-mark',
+					},
+				},
+			},
+		},
+	},
+	'search': {
+		pattern: /(@search\s+|#)[a-zA-Z0-9\-.]+(?:\.\*)?(?=\s*;)/,
```

Comment:
Isn't the `#` part of the name? If yes, then this can be done like this:

```suggestion
		pattern: /(?:(@search\s+)|#)[a-zA-Z0-9\-.]+(?:\.\*)?(?=\s*;)/,
```

Reply:
`@pattern A = Word;` and `#A = Word;` - are pattern definitions, but
`@search A;` and `#A;` - are not pattern definitions, but are search targets.

Reply:
In the second case (`#A;`) the `#` is not part of the name. The idea behind `#` is to simplify one-string examples where the pattern is defined and used for search.